apps/frontend/src/pages/LoginPage/LoginPage.module.css

```diff
@@ -1,8 +1,38 @@
-.loginContainer {
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap');
+
+body {
+  color: #1d1d1f;
+}
```

Comment:
I don't think its safe to overwrite the `body` in here. You should do it in the index.css file.